.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,6 +1,6 @@
 ---
 name: Feature request
-about: Request an enhancement, not a bug fix
+about: Thanks, but we're not adding new features
```

Comment:
This already says this on line 7 below.

But now we'd also say it before they click 'New feature':
<img width="677" alt="Screenshot 2023-12-06 at 3 30 01 PM" src="https://github.com/studiopress/genesis-custom-blocks/assets/4063887/e0c125c9-1da1-43f1-bacf-31a52156cf82">